.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -90,4 +90,6 @@ will share this change with the people who need to know about it._
 
 ## [optional] What gif best describes this PR or how it makes you feel?
 
-![alt_text](gif_link)
+<!-- ![gif-name](link)  or -->
+<img src="https://media.giphy.com/media/2KAGlmkPywhZS/giphy.gif" width="120" height="auto"/>
```

Comment:
@KirtanSoni I think we may want the dead link here so that it makes it easy for folks to just copy and paste there gif link between the brackets